internal/cmd/rsl/annotate/annotate.go

```diff
@@ -38,7 +40,13 @@ func (o *options) Run(_ *cobra.Command, args []string) error {
 		return err
 	}
 
-	return repo.RecordRSLAnnotation(args, o.skip, o.message, true)
+	err = repo.RecordRSLAnnotation(args, o.skip, o.message, true)
+	if err != nil {
+		return err
+	}
+	slog.Info("Added RSL annotation")
```

Comment:
This could well be a statement in RecordRSLAnnotation and we'd need no diff here...